libs/search-widget/src/viewer/store.ts

```diff
@@ -447,9 +448,15 @@ function findParagraphFromSearchSentence(
 ): Paragraph | undefined {
   const field = resource.data[getFieldType(searchSenctence.field_type) as keyof ResourceData]?.[searchSenctence.field];
   const paragraphs = field?.extracted?.metadata?.metadata?.paragraphs;
+  const text = normalizeSearchParagraphText(searchSenctence.text);
   return paragraphs?.find((paragraph) =>
     strict
-      ? paragraph.sentences?.find((sentence) => searchSenctence.text === getSentenceText(field!, sentence))
-      : (getParagraphText(field!, paragraph) || '').includes(searchSenctence.text.trim()),
+      ? paragraph.sentences?.find((sentence) => text === getSentenceText(field!, sentence))
+      : (getParagraphText(field!, paragraph) || '').includes(text.trim()),
   );
 }
+
+function normalizeSearchParagraphText(text: string) {
+  return text.replace('<mark>','').replace('</mark>', '');
```

Comment:
This will replace only the first occurence of `<mark>` and `</mark>`. Is this in purpose?

Reply:
I didn't notice. It's fixed